src/Donations/index.tsx

```diff
@@ -176,6 +177,7 @@ function DonationInfo() {
           <BackButton color={"#000"} />
         </button>
       )}
+      {router.query.to && <div className={"topProjectBadge"}>Top Project</div>}
```

Comment:
I see this now for every project. I assume this is also still a draft?

Reply:
Yes, both of the PRs are in draft as the work in backend is still in progress.

Reply:
Converted both to draft pull requests (as provided as a feature by GitHub).